app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -273,7 +270,8 @@ def create_custom_button(input, model, record)
       :url_parms => "?id=#{record_id}&button_id=#{button_id}&cls=#{model}&pressed=custom_button&desc=#{button_name}"
     }
     button[:text] = button_name if input[:text_display]
-    button[:onwhen] = '1+' if cb_enabled_for_nested
+    button[:onwhen] = '1+' if cb_send_checked_list
+    button[:send_checked] = true if record_id == 'LIST'
```

Comment:
This line does nothing in `gaprindashvili`. (The equivalent functionality happens when `record_id == "LIST"`, based on the value of `url_parms`.)

(No harm though.)